src/content/configuration/optimization.md

```diff
@@ -200,6 +200,32 @@ module.exports = {
 };
 ```
 
+## `optimization.chunkIds`
+
+`bool: false` `string: natural, named, size, total-size`
+
+Tells webpack which algorithm to use when choosing chunk ids. Setting `optimization.chunkIds` to `false` tells webpack that none of built-in algorithms should be used, as custom one can be provided via plugin. By default `optimization.chunkIds` is set to `false`.
```

Comment:
From the current [code on master](https://github.com/webpack/webpack/blob/f2f62c76080dd34eb319a6031fe688b7a4716cf9/lib/WebpackOptionsApply.js#L413-L427), it looks like `optimization.chunkIds` defaults to `natural`.

It would also default to `named` in `development` and `total-size` in `production`, though that's only if the defaults for `optimization.namedChunks` and `optimization.occurenceOrder` aren't modified (which hopefully they wouldn't be since they were deprecated in [v4.16.0](https://github.com/webpack/webpack/releases/tag/v4.16.0)).